src/dsp_tools/utils/excel_to_json/utils.py

```diff
@@ -0,0 +1,289 @@
+# pylint: disable=wrong-import-order
+
+from __future__ import annotations
+
+from typing import Any
+import numpy as np
+import pandas as pd
+import regex
+from unittest import mock
+
+from dsp_tools.models.exceptions import UserError
+
+languages = ["en", "de", "fr", "it", "rm"]
+
+
+def read_excel_file(excel_filename: str) -> pd.DataFrame:
+    """
+    This function reads an Excel file, if there is a ValueError then it patches the openpyxl part that creates the
```

Comment:
I'm a big fan of [Semantic Line Breaks](https://sembr.org/). It has some huge advantages:

1) If you change something later, the diff is cleaner.
2) If you change something later, and you insert 2-3 words in the first line, you don't have to reorganize the entire paragraph.
3) For a writer, semantic line breaks allow the physical structure of text to reflect the logical structure of the thoughts that produce them.

That's why I use Semantic Line Breaks nearly everywhere. But of course, this is just a recommendation.